CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented here.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [Unreleased]
+
+### Added
+
+- Autocomplete now supports the same slots as AutocompleteField.
```

Comment:
Nice to see this got implemented! Any particular reason you prefer slot forwarding over the alternate methods of passing component params/writing a custom autocomplete?

Reply:
Not to say we prefer it this way, we just learnt that slot forwarding is possible and figured it didn't hurt to forward them where it was appropriate. As for writing a custom autocomplete, there are still many reasons to do so, which is why we will continue to provide subcomponents to mix-n-match at one's will.